Frontend/src/test/pages/marketplace/components/product/ProductReviews.test.jsx

```
@@ -0,0 +1,87 @@
+import { vi } from 'vitest';
+import { ProductReviews } from '../../../../../pages/marketplace/components/product/ProductReviews';
+
+// Mock lucide-react Star icon
+vi.mock('lucide-react', () => ({
+  Star: ({ className }) => <svg data-testid="Star" className={className} />
+}));
+
+// Mock UI components
+vi.mock('../../../../../components/ui/button', () => ({
+  Button: ({ children, onClick }) => (
+    <button data-testid="Button" onClick={onClick}>{children}</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)